android/sdk/src/main/java/com/taobao/weex/WXSDKInstance.java

```diff
@@ -625,6 +635,24 @@ public void reloadImages() {
     }
   }
 
+
+  public boolean isPreRenderMode() {
+    return this.isPreRenderMode;
+  }
+
+  public void setPreRenderMode(final boolean isPreRenderMode) {
+    WXSDKManager.getInstance().getWXRenderManager().postOnUiThread(new Runnable() {
+      @Override
+      public void run() {
+        WXSDKInstance.this.isPreRenderMode = isPreRenderMode;
+      }
+    },0);
+  }
+
+  public void replaceContext(@NonNull Context context) {
```

Comment:
'setContext' is fair enough. Pls add more comments about why context is set-able